spec/expression.dd

```diff
@@ -1255,6 +1256,41 @@ $(GNAME CastExpression):
         }
         ----
 
+$(H2 $(LNAME2 throw_expression, Throw Expression))
+
+$(GRAMMAR
+$(GNAME ThrowExpression):
+    $(D throw) $(GLINK AssignExpression) $(D ;)
+)
+
+$(P
+    $(I AssignExpression) is evaluated and must be a `Throwable` reference. The
+    `Throwable` reference is thrown as an exception, interrupting the current
+    control flow to continue at a suitable $(D catch) clause of a
```

Comment:
Mention `scope(failure)`?

Reply:
The paragraph now mentions `scope (exit|failure)` and links to the corresponding documentation in `statement.dd`.
(Waiting for the CI to check whether the link is correct)